### DISCUSSION TITLE
Copyright problems when showing Rnote in a video

### DISCUSSION THREAD

**Discussion by KostyaOs** [flxzt/rnote#860] [Q&A] [answered]
Hi!
I want to make a video lesson on a biology topic and in that video I intend to show Rnote with my notes for the lesson. Some lessons are free and some are paid. Is there a need to mention Rnote anywhere in videos like that or are there any problems with copyright in this situation? I checked GNU License but couldn’t find anything about this situation.

**Comment by flxzt** [comment, marked as answer]
Hey! This shouldn't be a problem at all. I am not a lawyer, but afaik the GPL only restricts redistributing the software itself commercially, it does not apply to copyright for content that is created with it.
Mentioning the app would of course be appreciated, but it's not necessary.

**Reply by KostyaOs** [reply]
Alright, thanks!
